install.rst

```diff
@@ -336,6 +336,19 @@ If you want to have a visual overview of your API in your browser you can add sw
 
 With this you can see the swagger-ui in your browser on port 8080.
 
+Deploying to Heroku
+===================
+Assuming your making modifications locally and then pushing to GitHub, it's easy to deploy to Heroku.
+ 
+1. Create a new app on Heroku
+2. In Settings add the following buildpack :code:`https://github.com/PostgREST/postgrest-heroku`
+3. Add the require Config Vars in Heroku (see above)
+4. Modify your postgres.conf file as required to match your Config Vars in Heroku
```

Comment:
I think it'd be better call the config `postgrest.conf`(extra `t`) we call it like that in other places in the docs.

Reply:
Is this what you had in mind?